src/error.js

```diff
@@ -45,8 +46,10 @@ export function normalizeError(error) {
     const data = get(error, 'response.data', null);
     const message = get(data, 'message') || get(error, 'message', null);
     const code = get(data, 'code') || get(error, 'response.status') || get(error, 'code', null);
+    const req = get(error, 'request', null);
+    const headers = req && req.getHeaders ? req.getHeaders() : null;
```

Comment:
This will be used by the `nodule-graphql` library when formatting errors before sending them on in the HTTP response